### PR TITLE
Update pkix key parser

### PR DESCRIPTION
**Pull request by treyridley**
Update the private key parser to accept both pkcs1 and pkcs8 formatted private keys.

### CODE REVIEW

File: pkg/attestlib/pkix_key.go [hunk 1]
```diff
@@ -33,12 +33,28 @@ func parsePkixPrivateKeyPem(privateKey []byte) (interface{}, error) {
 	if len(rest) != 0 {
 		return nil, errors.New("expected one public key")
 	}
-
-	key, err := x509.ParsePKCS8PrivateKey(der.Bytes)
-	if err != nil {
-		return nil, err
+	switch der.Type {
+	case "RSA PRIVATE KEY":
```
Comment (alexcope) [inline pkg/attestlib/pkix_key.go:37]: extract these out as constants

Comment (treyridley) [reply pkg/attestlib/pkix_key.go]: done.

File: pkg/attestlib/pkix_key.go [hunk 2]
```diff
@@ -27,18 +27,38 @@ import (
...
 	}
-
-	key, err := x509.ParsePKCS8PrivateKey(der.Bytes)
-	if err != nil {
-		return nil, err
+	switch der.Type {
+	case rsaPkcs1Key:
+		key, err := x509.ParsePKCS1PrivateKey(der.Bytes)
+		if err != nil {
+			return nil, err
```
Comment (ooq) [inline pkg/attestlib/pkix_key.go:44]: NIT: can you wrap the error? e.g., `errors.Wrap(err, fmt.Sprintf("parse %s key failed", der.Type))`. Same for below.

**Comment by ooq** [comment]
Thanks for the PR! Approved.
IDK why the cloud build tests failed. Can you push again to trigger a rerun? (I can manually trigger rerun but I found that manual rerun results would not propagate). 
Ping me if the rerun still fails.

**Comment by ooq** [comment]
@treyridley Can you just try to push some change to retrigger the cloud build test? Might just be something flaky.

File: pkg/attestlib/pkix_key.go (hunk 2 above)
Comment (treyridley) [reply pkg/attestlib/pkix_key.go]: done.

**Pull request merged**